### PR TITLE
Add Calculator component

### PR DESCRIPTION
**Pull request by niicode**
In this milestone I was able to add the following:

- Added initial ui for the calculator component
- Added styling to the calculator component

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (DammyShittu) [inline src/App.js]: - [x] All components should be class-based. Kindly refactor your app component to meet this requirement.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @niicode,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] You are expected to delete boiler plate files that will not be used, to keep your working directory clean. Kindly remove the marked files from the screenshots.

  public dir | src dir
  :---:|:---:
  app | math-com


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (niicode) [reply src/App.js]: @DammyShittu Awesome feedback, cheers 🎉

**Review by omarsalem7 — APPROVED** [review]
Hi @niicode 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**